ateam_kenobi/src/skills/blockers.cpp

```diff
@@ -101,6 +105,6 @@ std::vector<Robot> Blockers::getRankedBlockableRobots(const World & world)
 ateam_geometry::Point Blockers::getBlockingPosition(const World & world, const Robot & blockee)
 {
   return blockee.pos +
-         ((kRobotDiameter + 0.1) * ateam_geometry::normalize(world.ball.pos - blockee.pos));
+         ((kRobotDiameter * 2.5) * ateam_geometry::normalize(world.ball.pos - blockee.pos));
```

Comment:
Magic number??

Reply:
Definitely a magic number we picked from testing and it just determines the distance the blocker robot should keep from the opponent.

Reply:
Also probably another good piece of logic to look at updating/understanding in that case